### DISCUSSION TITLE
Statistics of vaccinated x unvaccinated

### DISCUSSION THREAD

**Discussion by adahlo** [msusicky/ockovani-covid#738] [Ideas]
Hi,
have you thought about adding the possibility to show the statistics of vaccinated vs. unvaccinated for past weeks? 
Currently it is possible to see just the actual week. There is no possibility to check again the history data.

Thanks!

**Comment by jan-stanek** [comment]
Hi, thanks for the idea, we've an issue (#705) for adding some chart with historical data.
